app-crypt/libscrypt/libscrypt-1.21-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs multilib-minimal
```

Comment:
Please sort.

Reply:
If multiple eclass EXPORT_FUNCTIONS the same phase function(e.g. src_configure), the last inherited one wins.
So I put mulitlib-minimal the last to remind follow-up maintainers to be caution when involving new eclasses.

And, https://github.com/gentoo/gentoo/blob/master/eclass/multilib-minimal.eclass#L16 says it should be the last in inherit order.

But, it's okay to sort, as toolchain-funcs has no exports in this case. Please confirm. 😃

Reply:
Then ok, leave as is, i haven't read multilib-minimal :)